### PR TITLE
Create html simple template file

### PR DESCRIPTION
**Pull request by serhii73**

### CODE REVIEW

**Comment by serhii73** [comment]
@webknjaz all good?

File: blog/templates/blog/post_list.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<html>
+    <head>
+        <title>Django Girls blog</title>
```
Comment (webknjaz) [inline blog/templates/blog/post_list.html]: please use i18n, as usual.

**Comment by serhii73** [comment]
@webknjaz I fixed `i18n`.

File: blog/templates/blog/post_list.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+</head>
+<body>
+<div>
+    <h1><a href="/">{% trans "Django Girls Blog" %}</a></h1>
+</div>
+
+<div>
+    <p>{% trans "published: 14.06.2014, 12:14" %}</p>
+    <h2><a href="">{% trans "My first post" %}</a></h2>
+    <p>{% trans "Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum. Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus." %}</p>
```
Comment (webknjaz) [inline blog/templates/blog/post_list.html]: Fun fact: there's a special tag for generating fish-text in template engines :)

File: blog/templates/blog/post_list.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{% load i18n %}
+
+<html>
+<head>
+    <title>{% trans "Django Girls blog" %}</title>
```
Comment (webknjaz) [inline blog/templates/blog/post_list.html]: I hope you'll introduce template inheritance in upcoming PRs, thus I won't require it now.

File: blog/templates/blog/post_list.html [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+</head>
+<body>
+<div>
+    <h1><a href="/">{% trans "Django Girls Blog" %}</a></h1>
+</div>
+
+<div>
+    <p>{% trans "published: 14.06.2014, 12:14" %}</p>
```
Comment (webknjaz) [inline blog/templates/blog/post_list.html]: Date part is variable, so it shouldn't affect static localizable part. Well, there's actually a way of having i18n in code parts, but we'll get to it eventually :)

**Comment by webknjaz** [comment]
> Create html simple template file

I would rephrase this as "Add post list template". Does this provide the reader more relevant information?

**Comment by serhii73** [comment]
Yes, `Add post list template` better than `Create html simple template file`, thank you.

File: blog/templates/blog/post_list.html (hunk 2 above)
Comment (serhii73) [reply blog/templates/blog/post_list.html]: https://docs.djangoproject.com/en/2.0/ref/templates/builtins/#lorem
